src/modules/transaction/hooks/usePriorityFee.js

```diff
@@ -0,0 +1,20 @@
+import { transactions } from '@liskhq/lisk-client';
+import { fromTransactionJSON } from '../utils/encoding';
+import { FEE_TYPES } from '../constants';
+
+const usePriorityFee = ({ selectedPriority, transactionJSON, paramsSchema }) => {
+  const feePerByte = selectedPriority.value;
+
+  try {
```

Comment:
I'm not a fan of using `try-catch` for an error that we can prevent. This function may fail if the transaction is invalid, or the schema or not right. You have all the means to prevent both cases. 
Using a try catch causes the function to run, consume the time, and the fail in an oblivion fashion. You don't know why it fails, you just know it failed. Even worse, we are not even using the exception message to had the error gracefully.

Reply:
true I have updated this